react/features/base/util/configUtil.js

```diff
@@ -0,0 +1,85 @@
+/* @flow */
+
+declare var config: Object;
+
+/**
+ * Defines some utility methods that are used before the other JS files are
+ * loaded.
+ */
+
+
+/**
+ * Builds and returns the room name.
+ *
+ * @returns {string}
+ */
+export function getRoomName(): string | typeof undefined {
```

Comment:
With config.js, interface_config.js, and logging_config.js at play I'd think that a configuration-related features/base will be a good and, consequently, necessary approach.